boilerplate/.gitignore

```diff
@@ -95,3 +95,8 @@ web-build/
 !.yarn/releases
 !.yarn/sdks
 !.yarn/versions
+
+# Dependency Graph Visualizations
+app-dependency-graph.dot
```

Comment:
do we need .dot in the repo level one?

Reply:
Yeah we can get rid of this. Done in [chore(boilerplate): update .gitignore to exclude app-dependency-graph](https://github.com/infinitered/ignite/pull/2999/commits/0291ebf672499e230b075fedde7a3db99ad8282e)